extensions/amp-list/0.1/test/validator-amp-list.html

```diff
@@ -126,6 +126,11 @@
     src="https://data.com/articles.json?ref=CANONICAL_URL"
     [is-layout-container]="containerState"></amp-list>
   </amp-list>
+  <!-- Valid amp-list with [diffable] attribute -->
```

Comment:
nit, use quote or quotes instead of brackets since brackets may be amp-bind attributes and this isn't one of those: `<-- Valid amp-list with 'diffable' attribute -->`

Reply:
Removed the brackets to be consistent with the rest of file. 👍